blueman/bluez/Adapter.py

```diff
@@ -16,27 +14,6 @@ class Adapter(Base):
     def _init(self, obj_path=None):
         super(Adapter, self)._init(self._interface_name, obj_path=obj_path)
 
-        self._object_manager = Gio.DBusObjectManagerClient.new_for_bus_sync(
-            Gio.BusType.SYSTEM, Gio.DBusObjectManagerClientFlags.NONE,
-            'org.bluez', '/', None, None, None)
-
-    def find_device(self, address):
```

Comment:
This is moved to Manager, but the usages are not updates, so they still expect `Adapter` to have `find_device`.